scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,94 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+Version 3 of etcd brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.  OpenShift 3.5 introduced etcd version 3.1.
+
+Version 3 of etcd implements a backwards compatible storage API that facilitates a two-step migration of the on-disk etcd database.  For migration purposes, the storage mode used by etcd 3.1 in OpenShift 3.5 remained in v2 mode.  As of OpenShift 3.6, new installs will use storage mode v3.  Upgrades from previous versions of OpenShift will NOT automatically migrate data from v2 to v3.  This must be done using supplied playbooks following the documented process.
```

Comment:
I'd say `For migration purposes, the storage mode used by OpenShift 3.5 was v2 mode even if etcd 3.x packages were used.` because we don't actually do anything to bound the etcd package version so they may have 3.0, 3.1, or soon 3.2.